airbyte_cdk/sources/declarative/auth/oauth.py

```diff
@@ -122,18 +136,27 @@ def get_token_refresh_endpoint(self) -> Optional[str]:
             return refresh_token_endpoint
         return None
 
+    def get_client_id_name(self) -> str:
+        return self._client_id_name.eval(self.config)  # type: ignore # eval returns a string in this context
+
     def get_client_id(self) -> str:
         client_id: str = self._client_id.eval(self.config)
         if not client_id:
             raise ValueError("OAuthAuthenticator was unable to evaluate client_id parameter")
         return client_id
 
+    def get_client_secret_name(self) -> str:
+        return self._client_secret_name.eval(self.config)  # type: ignore # eval returns a string in this context
+
     def get_client_secret(self) -> str:
         client_secret: str = self._client_secret.eval(self.config)
         if not client_secret:
             raise ValueError("OAuthAuthenticator was unable to evaluate client_secret parameter")
         return client_secret
 
+    def get_refresh_token_name(self) -> str:
```

Comment:
💅 Should these be `_key` instead?
```suggestion
    def get_refresh_token_key(self) -> str:
```

Reply:
The naming was taken out from the existing methods, like these ones:
https://github.com/airbytehq/airbyte-python-cdk/blob/main/airbyte_cdk/sources/declarative/auth/oauth.py#L143-L147

Should we change them all in this case? I'd have it as `_name` still to have consistency, but I don't mind having `_key` if this is critical.

Reply:
Fair enough! didnt see that was already the convention in the cdk.